### DISCUSSION TITLE
gh pages deploy error

### DISCUSSION THREAD

**Discussion by sonatou** [community/community#32805] [Other Feature Feedback, Questions, & Ideas] [answered]
i'm having a pipeline error when trying to deploy my gh page.

```
Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
No such file or directory @ dir_chdir - /github/workspace/docs
```

this is the repository 
https://github.com/sonatou/React.net-BandLister/tree/master

Can someone help me?

**Comment by airtower-luna** [comment, marked as answer]
Most likely you set a directory that doesn't exist as the publishing source, but it's hard to be sure without seeing the actual log.

**Reply by sonatou** [reply]
how can i send you that?

**Reply by airtower-luna** [reply]
Now that you added the link to the repository it's easy enough to find by clicking on the "Actions" tab. Thanks!

**Reply by airtower-luna** [reply]
And I see you changed the directory to deploy from `docs/` to the repository root already and now it works. So my suspicion was correct. :grin:

**Reply by sonatou** [reply]
yeahh, it taked a while to deploy, but it worked, THANKS A LOT for the help tho ! :D

**Reply by airtower-luna** [reply]
Glad I could help! Marking the answer would be appreciated. :slightly_smiling_face:

**Reply by adcoding** [reply]
So what did you changed to make it work? I'm having the same issue.

**Reply by airtower-luna** [reply]
If the error message is the same, most likely you'll also need to change the directory to deploy from (from `docs/` to `/`).

**Reply by arisdelacruz** [reply]
I'm also getting the same error message. Here's the repo:

https://github.com/arisdelacruz/segismundo.github.io

Thanks.

**Comment by Ralph-Gacad** [comment]


**Reply by radar-blip** [reply]
Rename `module2-solution:index.html`
to just `index.html`
that might fix it.

**Reply by Ralph-Gacad** [reply]
It worked!! Thank you so much!!

**Reply by radar-blip** [reply]
Anytime. Glad to help! 👍🏻

**Reply by IvanSliuzas** [reply]
> Cambiar el nombre `module2-solution:index.html` a solo `index.html` eso podría solucionarlo.

Genioooooooo!!!!!

**Comment by NaveeshGaonkar** [comment]
i'm having a pipeline error when trying to deploy my gh page.

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
No such file or directory @ dir_chdir - /github/workspace/docs


https://github.com/NaveeshGaonkar/Gaonkar_Naveesh.github.io

can   any one help me on this

**Reply by adcoding** [reply]
I'm having the same issue. Did you solve it?

**Reply by NaveeshGaonkar** [reply]
Nope

**Reply by adcoding** [reply]
I fixed by editing docs with root in the gh page configuration. Put root instead.
![image](https://user-images.githubusercontent.com/50012804/218990225-451c2cd0-fe5a-41c0-aa2e-d003d7faf332.png)

**Reply by AustinDeng** [reply]
I met the same problem, but solved by the method you provided.Nice!!! @adcoding

**Comment by IvanSliuzas** [comment]
Cambie el nombre del archivo a "index"

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by adcoding** [reply]
Cual archivo? English please

**Comment by IvanSliuzas** [comment]
Rename module2-solution:index.html
to just index.html
that might fix it.

Hay que aprender español...


Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by nssensalo** [comment]
My site only shows the readme and not my actual website. If I click on the localhost link in the readme it works fine. I am already on the root. What could be the problem? https://github.com/nssensalo/nssensalo.github.io

**Comment by 3117310** [comment]
Updated homepage makes it difficult to use iOS mobile browser上传IMG_2236。MP4...

**Comment by 3117310** [comment]
Uploading IMG_2236.MP4…
